infra/src/main/kotlin/fi/fta/geoviite/infra/configuration/RequestFilter.kt

```diff
@@ -89,7 +89,6 @@ class RequestFilter @Autowired constructor(
             details = UserDetails(UserName.of("HEALTH_CHECK"), null, null, null),
             role = Role(
                 code = Code("health-check"),
-                name = AuthName.of("Service Health Check"),
```

Comment:
Tämän olemassaolo sai mut hieman miettimään että olisiko roolin nimikenttä sittenkin pitänyt jättää (nimikenttää ei oltu käytetty missään frontilla mutta se oli silti jätetty kantaan pyörimään - tarkoituksella vai vahingossa?) En tosin keksi sille mitään muita käyttötarpeita kuin mahdolliset lokitukset, ja niistä me varmaan löydetään roolit paremmin koodin kuin nimen perusteella.

Reply:
Joo, tuohan siinä vähän oli alkuperäisenä ajatuksena että sillä olisi selkokielinen nimi muuallakin kuin frontissa. Mutta eipä tuolle mitään tarvetta ole koska koodikin on selkokielinen. Eli hyvä näin :+1: